docs/validate.md

```diff
@@ -0,0 +1,32 @@
+## Using the Validation Command
+
+The validation command runs a series of checks on a clean Git repository to ensure that the current state of a repository is valid.
+
+### Usage 
+The command accepts either a local (l or local) or a remote (r or remote) flag. These are used to determine whether to validate local or remote charts.
+
+1. **Mode Verification**: The command first establishes whether you're operating in remote or local mode.
+
+2. **Script Options Parsing**: It proceeds to decode your script options file, making sense of all the parameters specified.
```

Comment:
Can you give some details about script options file ? where is it located and how does the charts repo use it ?